src/_apis/crew/crew.ts

```
@@ -23,3 +25,18 @@ export async function createCrew(data: CreateCrewRequestTypes) {
+export async function editCrew(id: number, data: EditCrewRequestTypes) {
+  try {
+    await fetchApi(`/api/crews/${id}`, {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      credentials: 'include', // Include authentication credentials
+      body: JSON.stringify(data),
+    });
+  } catch (error) {
+    throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)